src/dune_rules/mdx.ml

```diff
@@ -308,14 +317,20 @@ let gen_rules_for_single_file stanza ~sctx ~dir ~expander ~mdx_prog
         (*The old mdx stanza calls the [ocaml-mdx] executable, new ones the
           generated executable *)
         let open Command.Args in
+        let color_always =
+          if Lazy.force Ansi_color.stderr_supports_color then
+            [ A "--color=always" ]
```

Comment:
Which mdx version was this added in?

Reply:
This has been in mdx since the beginning.